### PR TITLE
Release 1.4

### PR DESCRIPTION
**Pull request by justinegeffen**

### CODE REVIEW

File: docs/installation/install-zoom-plugin.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+description: Get the Mattermost plugin installed as a first step
+---
+
+# Install Zoom Plugin
+
+### Install via Plugin Marketplace \(Recommended\)
+
+1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost
```
Comment (justinegeffen) [inline docs/installation/install-zoom-plugin.md:9]:
```suggestion
1. Go to **Main Menu > Plugin Marketplace** in Mattermost
```

File: docs/installation/zoom-configuration/webhook-configuration.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Webhook Configuration
+
+## Configure Webhook Events
+
+When a meeting finishes in Zoom, the original link shared in the channel can be changed to indicate the meeting has ended and how long it lasted. To enable this functionality, we need to create a webhook subscription in Zoom that tells the Mattermost server every time a meeting ends. Then Mattermost server can update the original Zoom message.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/webhook-configuration.md:5]:
```suggestion
When a Zoom meeting ends, the original link shared in the channel can be changed to indicate the meeting has ended and how long it lasted. To enable this functionality, we need to create a webhook subscription in Zoom that tells the Mattermost server every time a meeting ends. The Mattermost server then updates the original Zoom message.
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+description: Steps to configure your Zoom account to work with Mattermost
+---
+
+# Zoom Setup \(OAuth\)
+
+You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom and use it to create meetings and pull user data.
+
+## Create an App for Mattermost
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:9]:
```suggestion
## Create an app for Mattermost
```

File: docs/usage/start-meetings.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+## Usage
+
+Once enabled, clicking the video icon in a Mattermost channel invites team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
+
+![](../.gitbook/assets/42196048-af54d2b8-7e30-11e8-80a0-5e160ae06f03.png)
+
+## Slash Command
+
+You can also start a meeting in any chat window by typing `/zoom start`
```
Comment (justinegeffen) [inline docs/usage/start-meetings.md:11]:
```suggestion
You can also start a meeting in any chat window by typing `/zoom start`.
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Near the end of the page, you will find a section called **Deauthorization Notification**.
+* Enter a valid **Endpoint URL** \(`https://SITEURL/plugins/zoom/deauthorization?secret=WEBHOOKSECRET`\). 
+  * `SITEURL` should be your Mattermost server URL.
+  * `WEBHOOKSECRET` is generated during [Mattermost Setup](../mattermost-setup.md).
+
+![Deauthorization Notification section](../../.gitbook/assets/screenshot-from-2020-06-05-20-04-33.png)
+
+## Finish Setting up Mattermost Server
+
+* Follow the instructions for [Mattermost Setup](../mattermost-setup.md)
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:52]:
```suggestion
* Follow the instructions for [Mattermost Setup](../mattermost-setup.md).
```

**Comment by justinegeffen** [comment]
I've added @amyblais for editorial review as there are a lot of content changes.

**Review by larkox — APPROVED** [review]
LGTM

File: docs/installation/install-zoom-plugin.md [hunk 6]
```diff
@@ -6,12 +6,16 @@ description: Get the Mattermost plugin installed as a first step
...
+2. Search for "Zoom" or manually find the plugin from the list and click **Install**.
+3. After the plugin has downloaded and been installed, follow the instructions on [Zoom Configuration](zoom-configuration/).
+4. After finishing the [Zoom Configuration](zoom-configuration/), follow the instructions on [Mattermost Setup](mattermost-setup.md).
 
 ### \(Alternative\) Install via Manual Upload
 
-If your server doesn't have access to the internet, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console &gt; Plugin Management.** The binary releases on the page above, are the same as used by the Marketplace.
+If your server doesn't have internet access, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console > Plugin Management**. The binary releases on the page above, are the same as used by the Marketplace.
 
+1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost
```
Comment (amyblais) [inline docs/installation/install-zoom-plugin.md:18]: These should have periods similar to the bullet points above.

File: docs/installation/zoom-configuration/README.md [hunk 7]
```diff
@@ -1,19 +1,24 @@
...
+   * Users can connect their Mattermost/Zoom accounts even if their emails do not match.
 2. [JWT/Password](zoom-setup-jwt.md)
-   * Users don't have to connect their account to use the integration which makes it easy to get started.  
-   * Uses JWT to pass security tokens.  This may not be sufficiently secure for some customers.
+   * Users don't have to connect their account to use the integration which makes it easy to get started.
+   * Uses JWT to pass security tokens. This may not be sufficiently secure for some customers.
+
+## Upgrading From a Previous Version
 
+If you've been using Zoom prior to v1.4 you likely have a legacy webhook-type app configured in Zoom.
```
Comment (amyblais) [inline docs/installation/zoom-configuration/README.md:20]:
```suggestion
If you've been using Zoom prior to v1.4, you likely have a legacy webhook-type app configured in Zoom.
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 8]
```diff
@@ -4,48 +4,45 @@ description: Steps to configure your Zoom account to work with Mattermost
 
 # Zoom Setup \(OAuth\)
 
-We are going to set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom and used to create meetings and pull user data:
+You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom and use it to create meetings and pull user data.
```
Comment (amyblais) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:7]:
```suggestion
You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and use it to create meetings and pull user data.
```

**Review by amyblais — APPROVED** [review]

**Pull request merged**